pkg/frontend/v2/frontend_test.go

```diff
@@ -142,7 +142,7 @@ func TestFrontendBasicWorkflow(t *testing.T) {
 	})
 
 	req := &httpgrpc.HTTPRequest{
-		Url: "/api/v1/query_range?start=946684800&end=946771200&step=60",
+		Url: "/api/v1/query_range?start=946684800&end=946771200&step=60&query=up{}",
```

Comment:
@francoposa not entirely sure why this worked before without a query, but I'm pretty sure they _should_ all have one.

Reply:
pretty sure it's because flag was off for that test.
The helper method parses the entire query using promql parser, which won't accept a missing a `query_range` request with no `query` param.

Reply:
Yeah, I didn't think it was off before my changes because of this line: https://github.com/grafana/mimir/blob/4bce24dc690d9939e968491ad8f5c1328a1263e3/pkg/frontend/v2/frontend_test.go#L71

But I see now that the line immediately following disregards any config already set in `cfg` and sets everything to defaults, which is a little confusing.

Reply:
yeah I guess I missed or didn't understand what that line did